src/main/java/fr/minuskube/inv/util/ItemBuilder.java

```diff
@@ -0,0 +1,275 @@
+        lore.add(line);
+        meta.setLore(lore);
+        return this;
+    }
+
+    /**
+     * Removes a line from the current lore using {@link List#remove(Object)}
+     *
+     * @param line The line of lore, that should be removed
+     * @return <code>this</code> instance
+     */
+    public ItemBuilder removeLore(String line) {
+        List<String> lore = meta.getLore();
+        if (lore != null) {
+            lore.remove(line);
+            meta.setLore(lore);
+        }
+        return this;
+    }
+
+    /**
+     * Sets a new amount for this item stack
+     *
+     * @param amount The new amount
+     * @return <code>this</code> instance
+     */
+    public ItemBuilder amount(int amount) {
+        stack.setAmount(amount);
+        return this;
+    }
+
+    /**
+     * Applies the given color to the item if the item is leather armor, so it MUST be one of the following
+     * <ul>
+     *     <li>LEATHER_HELMET</li>
+     *     <li>LEATHER_CHESTPLATE</li>
+     *     <li>LEATHER_LEGGINGS</li>
+     *     <li>LEATHER_BOOTS</li>
+     * </ul>
+     *
+     * @param color The new color that should be applied
+     * @return <code>this</code> instance
+     * @throws IllegalStateException This exception is thrown, if the material is NOT one of the listed types
+     */
+    public ItemBuilder leatherColor(Color color) {
+        switch (stack.getType()) {
+            case LEATHER_HELMET:
+            case LEATHER_CHESTPLATE:
+            case LEATHER_LEGGINGS:
+            case LEATHER_BOOTS:
+                LeatherArmorMeta lam = (LeatherArmorMeta) meta;
+                lam.setColor(color);
+                meta = lam;
+                break;
+            default:
+                throw new IllegalStateException("To be able to use ItemBuilder#color(c) the material MUST be either LEATHER_HELMET, LEATHER_CHESTPLATE, LEATHER_LEGGINGS or LEATHER_BOOTS");
+        }
+        return this;
+    }
+
+    /**
+     * Adds an enchantment directly to the item stack without level restrictions
+     *
+     * @param enchantment The enchantment to add
+     * @param level The level of the enchantment
+     * @return <code>this</code> instance
+     */
+    public ItemBuilder enchantment(Enchantment enchantment, int level) {
+        stack.addUnsafeEnchantment(enchantment, level);
+        return this;
+    }
+
+    /**
+     * Adds an enchantment with the level 1 to the item stack
+     *
+     * @param enchantment The enchantment to add
+     * @return <code>this</code> instance
+     */
+    public ItemBuilder enchantment(Enchantment enchantment) {
+        return enchantment(enchantment, 1);
+    }
+
+    /**
+     * Adds an enchantment to the item meta without level restrictions
+     *
+     * @param enchantment The enchantment to add
+     * @param level The level of the enchantment
+     * @return <code>this</code> instance
+     */
+    public ItemBuilder metaEnchantment(Enchantment enchantment, int level) {
```

Comment:
This method is not necessary, adding enchentment from ItemStack class is the same thing.